StochasticBarrierFunctions/src/barrier_algorithms/base.jl

```diff
@@ -0,0 +1,8 @@
+### Algorithm types
```

Comment:
This is a lot cleaner, thanks. I will update the toolbox description accordingly.

Reply:
- [ ] Setup website for documentation

Reply:
What do you mean? I only moved the algorithm struct definitions into their separate files.

Reply:
I am thinking about making the toolbox paper story more clear